README.md

```diff
@@ -6,9 +6,49 @@ Exposes Prometheus metrics from [Starboard][starboard-upstream]'s `Vulnerability
 
 ## Metrics
 
-This exporter exposes two types of metrics:
+This exporter exposes various types of metrics:
 
-### Summary
```

Comment:
I'd suggest we add some hierarchy now, since we also have ConfigAuditReports not shown here.
What about something like:
```
## Metrics
This exporter exposes several types of metrics:

### CIS Benchmarks
#### Report Summary
Exposes an overview of the entire report...
#### Section Summary
For slightly more granular visibility...
#### Result Details
Exposes detailed information about the results of each test...

### Vulnerability Reports
#### Report Summary
...
#### Vulnerability Details
```
One of us can add configauditreports later

Reply:
Done 🎸